### DISCUSSION TITLE
Is it expected that I see every activities _but_ the ones on my own repositories?

### DISCUSSION THREAD

**Discussion by ericmorand** [community/community#83148] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My home page displays the activities of many repositories _but_ the ones that I own or participate to.

Is this expected? If so, what is the rational of filtering out my own repositories?

**Comment by Shahnawazk7944** [comment]
**GitHub doesn't have a direct filter out feature specifically for our own repositories within the Overview section.**

Read Doc -> https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile

**_consider submitting a feature request to GitHub: https://github.com/contact_**

**Comment by ericmorand** [comment]
@Shahnawazk7944 so, there is no way to see what our colleagues have been doing on the repositories that we work together on?

And what happens if I join an organisation that I follow? Does it suddenly disappear from the home page?
